eg/Classes/PluginBase.py

```diff
@@ -148,7 +148,15 @@ def AddActionsFromList(self, theList, defaultAction=None):
         self.info.actionGroup.AddActionsFromList(theList, defaultAction)
 
     def AddEvents(self, *eventList):
-        self.info.eventList = eventList
+        if self.info.eventList is None:
+            self.info.eventList = ()
+
+        for event in eventList:
+            if isinstance(event, basestring):
+                event = (event, '')
+            elif isinstance(event, list):
+                event = tuple(event)
+            self.info.eventList += (event,)
```

Comment:
i meant to say have the EventItem just pass it's name to the AddEventList and the AddEventList will check to see if it exists and if not it will add it